ideascube/serveradmin/catalog.py

```diff
@@ -126,6 +129,10 @@ def remove(self, package):
         self._removed.append(package)
 
     def commit(self):
+        visible_packages = get_config('ideascube', 'displayed-package-ids')
+        [visible_packages.remove(p.id) for p in self._removed if p.id in visible_packages]
+        visible_packages.extend(p.id for p in self._installed)
+        set_config('ideascube', 'displayed-package-ids', visible_packages, User.objects.get_system_user())
```

Comment:
The last line is too long.

Also, eventually we might want to use the actual `user` who installed the packages instead of the system user. But that will only make sense when we have a UI to install packages from ideascube (right now we can only use the CLI) so we can cross that bridge when we get there. Something to keep in mind, though.